lineman/app/services/key_event_service.coffee

```diff
@@ -12,15 +12,21 @@ angular.module('loomioApp').factory 'KeyEventService', ($rootScope) ->
       40:  'pressedDownArrow'
 
     broadcast: (event) ->
-      if !event.ctrlKey and !event.altKey and key = @keyboardShortcuts[event.which]
+      if key = @keyboardShortcuts[event.which]
         $rootScope.$broadcast key, event, angular.element(document.activeElement)[0]
 
     registerKeyEvent: (scope, eventCode, execute, shouldExecute) ->
       shouldExecute = shouldExecute or @defaultShouldExecute
       scope.$on eventCode, (angularEvent, originalEvent, active) ->
-        if shouldExecute(active)
+        if shouldExecute(active, originalEvent)
           angularEvent.preventDefault() and originalEvent.preventDefault()
           execute(active)
 
-    defaultShouldExecute: (active = {}) ->
-      !_.contains ['INPUT', 'TEXTAREA', 'SELECT'], active.nodeName
+    defaultShouldExecute: (active = {}, event = {}) ->
+      !event.ctrlKey and !event.altKey and !_.contains(['INPUT', 'TEXTAREA', 'SELECT'], active.nodeName)
+
+    submitOnEnter: (scope) ->
```

Comment:
Hey, so the way this works:

- For a form, mark some fields as lmo-primary-form-input
- In the form controller, mark the form as accepting acceptOnEnter events. (A better iteration of this might apply it to all forms, possibly through the FormService)
- User goes to the form, fills it out, hits cmd/ctrl+enter
- If the element in focus is a) a lmo-primary-form-input field, and b) is within the scope of the form, call form.submit (this is what's great about generalizing the name of the submit action)

I applied to vote, proposal, discussion, and comment forms, but it'll be easy to expand to more if we need it.